pkg/tsdb/cloudmonitoring/time_series_query.go

```diff
@@ -132,6 +130,14 @@ func (timeSeriesQuery cloudMonitoringTimeSeriesQuery) parseResponse(queryRes *ba
 			seriesLabels["metric.name"] = d.Key
```

Comment:
Every metric can have zero or many labels associated with them. When specified in the frame, they can for example be used to format the legend using [display name options](https://grafana.com/docs/grafana/latest/panels/standard-options/#display-name). 

I guess we could do something like this:
```go
if len(response.TimeSeriesDescriptor.PointDescriptors) > 1 && !strings.HasSuffix(d.Key, ".mean") {
	seriesLabels[d.Key] = ""
	continue
} ```
But not sure how meaningful it is since you wouldn't be able to see the value. It would be great if the value field on the frame could accept not only float64, but a map of key values. That way we could pass all aggregations.

Reply:
But the continuation should definitely be done before setting the labels